README.md

```diff
@@ -119,36 +121,106 @@ export default Total;
 
 ## API Documentation
 
+### createSelector(...inputSelectors, resultFn)
 ### createSelector([inputSelectors], resultFn)
 
-Takes an array of selectors whose values are computed and passed as arguments to resultFn.
+Takes a variable number or array of selectors whose values are computed and passed as arguments to `resultFn`. A selector has the signature `(state, props) => state`.
 ```js
 
 const mySelector = createSelector(
-  [
+  state => state.values.value1,
+  state => state.values.value2,
+  (value1, value2) => value1 + value2
+);
+
+// You can also pass an array of selectors
+const totalSelector = createSelector(
+  [ 
     state => state.values.value1,
     state => state.values.value2
   ],
   (value1, value2) => value1 + value2
 );
 
-// it is not necessary to wrap a single input selector in an array
-const totalSelector = createSelector(
-  state => state.shop.items,
-  items => items.reduce((acc, item) => acc + item.value, 0)
+// A selector is also passed props as the last parameter into the resultFunc
```

Comment:
Can this result in problems, as the selector suddenly depends on all properties.
Wouldn't it be better to have the following signature

createSelector(
  state, props => state.values.value1,
  state, props => state.values.value2,
  state, props => props.value3,
  value1, value2, value2 => value1, value2, value3
)